### PR TITLE
[ENG-5017] display queue type when waiting

### PR DESCRIPTION
**Pull request by dsokal**
# Checklist

- [ ] I've added an entry to [CHANGELOG.md](https://github.com/expo/eas-cli/blob/main/CHANGELOG.md#main) if necessary. You can comment this pull request with `/changelog-entry [breaking-change|new-feature|bug-fix|chore] [message]` and CHANGELOG.md will be updated automatically.

# Why

https://linear.app/expo/issue/ENG-5017/suggestion-display-queue-type-in-cli-when-waiting

# How

Display queue type when waiting.

# Test Plan
Screenshot 2022-07-18 at 13 55 41

### CODE REVIEW

File: packages/eas-cli/src/build/build.ts [hunk 1]
```diff
@@ -385,6 +385,12 @@ async function handleSingleBuildProgressAsync(
   return { refetch: true };
 }
 
+const priorityToQueueDisplayName: Record<BuildPriority, string> = {
+  [BuildPriority.Normal]: 'Low Priority Queue',
+  [BuildPriority.NormalPlus]: 'High Priority Queue (free upgrade)',
+  [BuildPriority.High]: 'High Priority Queue',
```
Comment (dsokal) [inline packages/eas-cli/src/build/build.ts:391]: 1. I'm not sure what looks better: uppercase or lowercase
2. I'm not sure if the wording for normal+ is correct. @brentvatne

Comment (kbrandwijk) [reply packages/eas-cli/src/build/build.ts:391]: This is different wording than we use on the website ('Standard Queue', 'Priority Queue').

Comment (dsokal) [reply packages/eas-cli/src/build/build.ts:391]: Good point, sth like this?
```suggestion
  [BuildPriority.Normal]: 'Standard Queue',
  [BuildPriority.NormalPlus]: 'Priority Queue (free upgrade)',
  [BuildPriority.High]: 'Priority Queue',
```

Comment (kbrandwijk) [reply packages/eas-cli/src/build/build.ts:391]: LGTM!

Comment (brentvatne) [reply packages/eas-cli/src/build/build.ts:391]:
```suggestion
  [BuildPriority.Normal]: 'queue',
  [BuildPriority.NormalPlus]: 'queue',
  [BuildPriority.High]: 'priority queue',
```

we don't want to surface the free upgrade bit at the moment, we just want those builds to be fast without that being advertised.

with the above suggestions, you'd see one of following two possible states:

> Waiting in queue...

> Waiting in priority queue...

(we don't use capitalized Priority on the pricing page)

**Review by brentvatne — APPROVED** [review]

**Comment by dsokal** [comment]
/changelog-entry new-feature Display build queue type when waiting.

**Pull request merged**